python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
As far as I can tell https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings says nothing about empty lines *before* docstrings, only about *after*.

But as I said, no strong feelings here, I'd just like to have a standard. Maybe we should also install pep257 on travis.